src/app.html

```diff
@@ -30,32 +30,251 @@
+			{
+				"@type": "HighSchool",
+				"name": "Landstown High School Governor's STEM and Technology Academy",
+				"alternateName": 
+				[
+					"Landstown",
+					"Landstown High School",
+					"Landstown High School STEM academy",
+					"Landstown HS",
+					"LHS",
+					"Landstown High",
+					"Landstown Eagles",
+					"Eagles",
+					"Landstown High School Tech Academy",
+					"Landstown High School STEM and Technology Academy",
+					"Governor's STEM Academy at Landstown High School"
+				],
+				"image": "https://www.bjuarchitects.com/wp-content/uploads/2016/02/front.jpg",
+				"sameAs": "https://en.wikipedia.org/wiki/Landstown_High_School",
+				"mainEntityOfPage": "https://landstownhs.vbschools.com/",
+				"address": "2001 Concert Drive Virginia Beach, VA 23456",
+				"naics": "611110",
+				"telephone": "(757) 648-5500",
+				"location": 
+				{
+					"@type": "Place",
+					"address": "2001 Concert Drive Virginia Beach, VA 23456",
+					"geo":
+					{
+						"@type": "GeoCoordinates",
+						"latitude": 36.7796828,
+						"longitude": -76.1006447,
+					},
+				},
+				"parentOrganization": 
+				{
+					"@type": "Organization",
+					"name": "Virginia Beach City Public Schools",
+					"image": "https://resources.finalsite.net/videos/t_video_vp9_1080/v1659644394/vbschoolscom/osxjcr2t4axfjrebs9t0/SchoolsCompilationWebsiteDraft2.webm",
+					"logo": "https://resources.finalsite.net/images/f_auto,q_auto,t_image_size_1/v1646127051/vbschoolscom/iribbwhg3k8eth5piggo/logo.png",
+					"sameAs": 
+					[
+						"https://en.wikipedia.org/wiki/Virginia_Beach_City_Public_Schools",
+						"https://schoolquality.virginia.gov/divisions/virginia-beach-city-public-schools"
+					],
+					"mainEntityOfPage": "https://www.vbschools.com/",
+					"address": "2512 George Mason Drive Virginia Beach, Virginia, 23456-0038 United States",
+					"location": 
+					{
+						"@type": "Place",
+						"address": "2512 George Mason Drive Virginia Beach, Virginia, 23456-0038 United States",
+						"geo":
+						{
+							"@type": "GeoCoordinates",
+							"latitude": 36.7517741,
+							"longitude": -76.0614749,
+						},
+					},
+					"naics": "611110",
+					"telephone": "(757) 263-1000",
+				},
+			},
+			"member": 
+			[{
+				"@type": "Person",
+				"@id": "https://github.com/LeoDog896",	
+				"name": "LeoDog896",
+				"alternateName": "Tristan F.",
+				"mainEntityOfPage": "https://leodog896.com/",
+				"sameAs": 
+				[
+					"https://github.com/LeoDog896",
+					"https://reddit.com/user/leodog896",
+					"https://replit.com/@LeoDog896",
+					"https://stackoverflow.com/users/7589775/leodog896"
+				],
+			},
+			{
+				"@type": "Person",
+				"@id": "https://github.com/willuhmjs",
+				"name": "William Faircloth",
+				"alternateName": 
+				[
+					"willuhmjs",
+					"willuhm-js"
+				],
+				"mainEntityOfPage": "https://willuhmjs.com",
+				"sameAs": 
+				[
+					"https://github.com/willuhmjs",
+					"https://rusecurectf.radford.edu/users/3421",
```

Comment:
This is not a social media account. Preferably, for me, @Daniel-Tomov, and @willuhmjs, leave only our main entity.

Reply:
I agree

Reply:
done